shared/graphql/schema.graphql

```diff
@@ -215,13 +215,24 @@ type User {
 type AccessCountForDate {
   date: Datetime!
   value: Int!
+  sum: Int!  # running sum
 }
 
 type AccessCounts {
   total: Int!
   counts: [AccessCountForDate!]!
 }
 
+type AccessCountsGroup {
+  ext: String!
+  counts: AccessCounts!
+}
+
+type BucketAccessCounts {
+  byExt(groups: Int): AccessCountsGroup[]
```

Comment:
don't merge / collapse the groups by default
(maybe we don't have to do that at all, just keep that logic on the front-end, not sure)

Reply:
I'm not quite sure what you mean

Reply:
[catalog code](https://github.com/quiltdata/quilt/blob/master/catalog/app/containers/Bucket/requests/requestsUntyped.js#L76) "collapses" access counts into a finite number of "bands", so if there's too many extensions, less frequent ones are merged into a single "other" band.
this proposed api design assumes there's no such collapsing performed by default (when `groups` are null or zero)